### ISSUE TITLE
I cannot play Castlevania Symphony of the Night, a Sony PlayStation (PSX) rom

### ISSUE THREAD

**Issue by buruto** [TASEmulators/BizHawk#3445]
I have a bin file of the rom whose size is 555 MB and a cue file of the rom whose size is 1 KB.

When I open the cue file there is unhandled exception error.
When I open the bin file I am asked to select the console or platform that this rom belongs to.

When I choose playstation and proceed I am asked for the firmware.
For playstation U I selected SCPH1001.bin bios and SCPH7502.bin for E.

Then I reload the rom and the title bar changes to show me that BizHawk is emulating the selected rom but the entire client of the BizHawk main window remains black all the time.

Is Castlevania Symphony of the Night playable on BizHawk at all?
If yes then what I am doing wrong?

**Comment by YoshiRulz** [comment]
Can you copy+paste the error you get when you open the `.cue`?

**Comment by buruto** [comment]
I can't because this is a message box thus both Ctrl+A or highlighting the text with my mouse and then copy won't work.

**Comment by YoshiRulz** [comment]
Ctrl+C with the dialog focused should work.

**Comment by buruto** [comment]
Should I open the cue file instead of the bin file?

**Comment by YoshiRulz** [comment]
The `.cue` references a file on the PC it was dumped on which doesn't exist on your PC. Open it in a text editor and change the absolute path to a relative one, making sure it matches the `.bin` filename.

**Comment by buruto** [comment]
Both the `.cue` and the `.bin` files are in the same folder and other playstation emulators like ePSXe, pcsx reloaded, no$psx and DuckStation can emulate the rom by selecting either `.cue` or `.bin`.

I am not required to edit any of them but if for BizHawk I must edit the `.cue` file then I will do it after making a backup.

**Comment by YoshiRulz** [comment]
I think you're right about other emulators "cheating", something we've been hesitant to do.

EmuHawk should be able to load the corrected `.cue`. It won't be exactly the same as loading the `.bin` because of the spaghetti code that handles rom loading, though if the `.bin` is corrupted it may have the same outcome.

**Comment by buruto** [comment]
Okay I made a backup of the original `.cue` file and then corrected the `.cue` file and selected the corrected `.cue`.

Strange that I was asked again for the firmware.
Again I chose SCPH1001.bin for U and SCPH7502.bin for E and reloaded the rom.

Finally BizHawk plays the rom! BUT the Debugger is greyed out and unavailable and I don't know why.

I want to use it to hack the game and make my own cheats but why I can't use the debugger?
